distribution/src/main/assembly/generic-bin.xml

```diff
@@ -103,6 +103,7 @@
         <include>commons-cli:commons-cli</include>
         <include>org.json:json</include>
         <include>org.apache.maven.shared:maven-shared-utils</include>
+        <include>net.sf.opencsv:opencsv</include>
```

Comment:
We need to add this to the notice.txt of the gateway and ee gateway now that we are using this